prog/kubernetes/kubernetes_node_nexus.rb

```diff
@@ -6,9 +6,20 @@ class Prog::Kubernetes::KubernetesNodeNexus < Prog::Base
   def self.assemble(project_id, sshable_unix_user:, name:, location_id:, size:, storage_volumes:, boot_image:, private_subnet_id:, enable_ip4:, kubernetes_cluster_id:, kubernetes_nodepool_id: nil)
     DB.transaction do
       id = KubernetesNode.generate_uuid
+
+      exclude_host_ids = []
+      unless kubernetes_nodepool_id
+        cluster = KubernetesCluster[kubernetes_cluster_id]
```

Comment:
We might move it to outer and remove `cluster = node.kubernetes_cluster` reassignment in the future lines